app/src/main/res/values/strings.xml

```diff
@@ -23,4 +23,7 @@
   <string name="speaker_list_title">Speakers</string>
   <string name="talks_list_title">Talks</string>
 
+  <string name="droidcon_hashtag" translatable="false">Droidconit</string>
```

Comment:
❌  why this change? The value comes from the configuration in `team-props`; please don't change things that are working correctly